sklearn_numba_dpex/kmeans/tests/test_kmeans.py

```diff
@@ -156,6 +156,7 @@ def test_euclidean_distance(dtype):
     estimator = KMeans(n_clusters=len(b))
     estimator.cluster_centers_ = b
     engine = KMeansEngine(estimator)
+    assert engine.accepts(a, y=None, sample_weight=None)
```

Comment:
I think we should add assertions when `y` or `sample_weight` are not `None`.

This applies on new assertions as well.